Dockerfile

```diff
@@ -30,5 +30,11 @@ FROM nginx:alpine
 
 COPY --from=builder /src/webapp /app
 
+# Insert wasm type into Nginx mime.types file so they load correctly.
+RUN sed '3i\ \ \ \ application/wasm wasm\;' /etc/nginx/mime.types > /tmp/mime.types
```

Comment:
any reason we can't pipe directly to the nginx config?

Reply:
I found out today that `sed` operates line by line, so if you `sed $thing a > a` you will typically end up with a blank file.

Reply:
you could `echo ... >> ...` (>> being append) though, no?

Reply:
Apparently you can even do `-i`. @babolivier is a wizard and he knows better than all of StackOverflow.

Reply:
brb printing that on a tee shirt.